content/04.discussion.md

```diff
@@ -1,26 +1,31 @@
 We detected and showed expected frequencies of genomic lesions, mutational signatures, and aberrantly regulated signaling pathways across multiple pediatric brain tumor histologies.
 Molecular subtyping information from pathology reports was largely not collected nor available for this cohort, and if available (e.g.: sparse medulloblastoma subtypes), it had to be manually curated from pathology reports and/or free text clinical data fields.
-Furthermore, in the absence of DNA methylation data, quick molecular subtyping classification could not be performed.
-Thus, to enable more targeted analyses to interrogate the biology of specific tumor subtypes, we created subtyping modules aligned with WHO molecularly-defined diagnoses.
-We worked closely with pathologists and clinicians to build modules from which we determined a research-grade integrated diagnosis for 44% (1260/2840) of biospecimens as well as identify incorrectly diagnosed or mis-identified samples in the OpenPBTA cohort. 
+Furthermore, in the absence of DNA methylation data, rapid classification to derive molecular subtypes could not be performed.
+Thus, to enable biological interrogation of specific tumor subtypes, we created RNA- and DNA-based subtyping modules aligned with WHO molecularly-defined diagnoses.
+We worked closely with pathologists and clinicians to build modules from which we determined a research-grade integrated diagnosis for 44% (1260/2840) of biospecimens as well as discovered incorrectly diagnosed or mis-identified samples in the OpenPBTA cohort. 
 
-We applied multiple previously validated classifiers to RNA expression data.
-We predicted medulloblastoma subtypes [@doi:10.1186/s13029-016-0053-y; @doi:10.1371/journal.pcbi.1008263], whether tumors have dysfunctional _TP53_ [@doi:10.1016/j.celrep.2018.03.076], and their degree of telomerase activity [@doi:10.1038/s41467-020-20474-9].
-By assessing _TP53_ and telomerase activity prospectively from expression data, information previously only attainable with genomic sequencing and/or qPCR, we can quickly add knowledge to our existing biological understanding of these tumors.
+We harnessed RNA expression data for a number of analyses, including application of multiple previously validated classifiers.
+From 43/122 (35.2%) medulloblastoma tumors whose pathology reports contained subtype information, we accurately predicted subtypes using MM2S (90.7%) or medulloPackage (95.4%) [@doi:10.1186/s13029-016-0053-y; @doi:10.1371/journal.pcbi.1008263].
+We used additional classifiers to determine whether tumors have dysfunctional _TP53_ [@doi:10.1016/j.celrep.2018.03.076] and their degree of telomerase activity from EXTEND scores [@doi:10.1038/s41467-020-20474-9].
+We found that hypermutant tumors universally display dysregulation of _TP53_ and that EXTEND scores are a robust surrogate measure for telomerase activity, as tumors with high scores have a significant adverse risk for overall survival. 
```

Comment:
Single tense:

```suggestion
We found that hypermutant tumors universally displayed dysregulation of _TP53_ and that EXTEND scores were a robust surrogate measure for telomerase activity, such that tumors with high scores had a significant adverse risk for overall survival. 
```

Reply:
I think we need to keep this active - this is really a call to action to use this data clinically